src/v/config/configuration.cc

```diff
@@ -701,7 +701,44 @@ configuration::configuration()
         model::fetch_read_strategy::polling,
         model::fetch_read_strategy::non_polling,
         model::fetch_read_strategy::non_polling_with_debounce,
+        model::fetch_read_strategy::non_polling_with_pid,
       })
+  , fetch_pid_p_coeff(
+      *this,
+      "fetch_pid_p_coeff",
+      "Proportional coefficient for fetch PID controller.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      100.0,
+      {.min = 0.0, .max = std::numeric_limits<double>::max()})
+  , fetch_pid_i_coeff(
+      *this,
+      "fetch_pid_i_coeff",
+      "Integral coefficient for fetch PID controller.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      0.01,
+      {.min = 0.0, .max = std::numeric_limits<double>::max()})
+  , fetch_pid_d_coeff(
+      *this,
+      "fetch_pid_d_coeff",
+      "Derivative coefficient for fetch PID controller.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      0.0,
+      {.min = 0.0, .max = std::numeric_limits<double>::max()})
+  , fetch_pid_target_utilization(
+      *this,
+      "fetch_pid_target_utilization",
+      "The overall precentage of reactor utilization the fetch scheduling "
```

Comment:
This should say "fraction between 0 and 1" not "precentage" since percentage is always between 0 and 100.

Reply:
Updated the description to make it clear that its a fraction, not a percentage.